ecommerce/serializers.py

```diff
@@ -1001,7 +1001,9 @@ def get_coupon(self, instance):
 
     def get_purchaser(self, instance):
         """Get the purchaser infrmation"""
-        return ExtendedLegalAddressSerializer(instance.purchaser.legal_address).data
+        data = ExtendedLegalAddressSerializer(instance.purchaser.legal_address).data
+        data.update({"company": instance.purchaser.profile.company})
```

Comment:
@umarmughal824 I guess, you need to move company inside the `ExtendedLegalAddressSerializer ` as I did in my PR

Reply:
this is not the part of the Legal Address model, that's why I have added it here.